docs/architecture.md

```diff
@@ -30,7 +29,7 @@ image that is built using standard Docker build processes. Elemental is
 built using normal `docker build` and if you wish to customize the OS
 image all you need to do is create a new `Dockerfile`.
 
-## rancherd
+## Rancher system agent
```

Comment:
The below test is a bit incomplete. We dont only have the rancher-sytem-agent but we have several components integrated into EleTeal (Not gonna call  it elemental teal all the time LOL) that take care of the bootstrap. So I would mention the register which is the one that registers the node and the operator which takes care of the bootstrapping and letting system-agent take care of the rke/k3s stuff.